### ISSUE TITLE
No in-game keyboard settings and red arrow signs on movement keys.

### ISSUE THREAD

**Issue by Rewizens** [michaelbarlow7/dungeon-crawl-android#81]
After updating to version 23.2 from 22.1 I faced an issue. Red arrows on movement keys are missing and there is no keyboard settings button in game. Can I have some help here please?

My the phone I am using is Huawei mate 9
Android 9.0.1

**Comment by joeljpa** [comment]
red arrow keys are missing for me...but seems cosmetic. 
No keyboard settings at all? I'm assuming you did try holding Recent apps softkey of your phone, then selecting "Preferences"?

**Comment by Rewizens** [comment]
Yes sure and red arrows option is enabled there, no arrows are visible on keyboard though

**Comment by charliegreen** [comment]
I've noticed that the arrows disappear depending on keyboard transparency. Try lowering the keyboard alpha. The option is available when you first long press to open the menu, before entering the preferences. I think this is a bug.
